ceramic-config/src/convert.rs

```diff
@@ -69,19 +69,13 @@ impl Into<crate::daemon::DaemonConfig> for crate::Config {
             admin_dids: Some(self.http_api.admin_dids),
             cors_allowed_origins: cors,
         });
-        let ipfs = Some(if let crate::Ipfs::Remote(r) = self.ipfs {
-            crate::daemon::DaemonIpfsConfig {
+        let ipfs = Some(crate::daemon::DaemonIpfsConfig {
                 mode: Some(crate::daemon::DaemonIpfsConfigMode::Remote),
-                host: Some(r.host),
+                host: Some("http://localhost:5101".to_string()),
```

Comment:
So this will work, but doesn't let users change the value. We will want instead to change

https://github.com/ceramicstudio/wheel/blob/main/ceramic-config/src/lib.rs#L41
https://github.com/ceramicstudio/wheel/blob/main/ceramic-config/src/lib.rs#L19

You also might want to change this prompt https://github.com/ceramicstudio/wheel/blob/main/cli/src/prompt/ceramic_advanced_config.rs#L57